compiler-js/src/test/ceylon/dyns/issues_dyns.ceylon

```diff
@@ -330,3 +331,9 @@ void issue5959() {
     check((d of Object) is C5959<String>, "#5959 targs3");
   }
 }
+
+void issue6057() {
+  dynamic DefaultObject {}
+  dynamic { DefaultObject o = eval("this"); }
+  check(identityHash(object extends Basic() {}) > 0, "#6057 1");
```

Comment:
@chochos: In 42f9321f6 (after the PR merge) it seems @gavinking changed the initial hash value from 1 to 0. I guess my test should have been updated to match that.